docs/src/apalache/principles/index.md

```diff
@@ -0,0 +1,19 @@
+# Principles of Symbolic Model Checking with Apalache
+
+In order to take advantage of Apalache's symbolic model checking, there are a
+few principles one must bear in mind when writing TLA.
+
+Note that Apalache requires type annotations.  Check the [Snowcat
+tutorial](../tutorials/snowcat-tutorial.md) and [HOWTO on
+annotations](../HOWTOs/howto-write-type-annotations.md).
```

Comment:
Feels like it could use more binding text (low priority though).

Reply:
yep, maybe later